newrelic/helpers.go

```diff
@@ -159,3 +159,7 @@ func revertEscapedSingleQuote(name string) string {
 
 	return name
 }
+
+func resourceDataWrapper(d *schema.ResourceData, key string) (interface{}, bool) {
+	return d.GetOk(key)
+}
```

Comment:
A better name for this function could be

```suggestion
func fetchAttributeValueFromResourceConfig(d *schema.ResourceData, key string) (interface{}, bool) {
	return d.GetOk(key)
}
```

or simply even

```suggestion
func fetchAttributeValue(d *schema.ResourceData, key string) (interface{}, bool) {
	return d.GetOk(key)
}
```

please also add a brief (one or two lines) description of why this is a necessary (useful) helper

Reply:
Updated name as suggested